### ISSUE TITLE
Add Reticulate Tutorial - `stumpyr`

### ISSUE THREAD

**Issue by seanlaw** [TDAmeritrade/stumpy#444]
Occasionally, users have asked if there was a way to use STUMPY in the R statistical software. It would be nice to have a simple tutorial that explained how to leverage the [Reticulate package](https://rstudio.github.io/reticulate/)

**Comment by Ninimama** [comment]
@seanlaw 
I would love to work on this tutorial.

**Comment by seanlaw** [comment]
@Ninimama It's all yours! I know very little about R

**Comment by Ninimama** [comment]
@seanlaw 
I worked on R for a course I had on advanced computational statistics. I am not a pro in R, but I would like to see if I can handle this.

**Comment by seanlaw** [comment]
Awesome! I know of some R folks who may be able to read/review the tutorial once it is ready

**Comment by Ninimama** [comment]
@seanlaw 

if you can, please make it available for others as I might not get enough time to go through this one. Sorry for the inconvenience. I will come back to it and check it out to see if someone picks it up or not.

**Comment by seanlaw** [comment]
Thank you for letting me know, @Ninimama! This is now open for anybody to work on

**Comment by seanlaw** [comment]
@RamiKrispin any chance you might be able to squeeze this into your 2022 open source contributions? 😅
